package.json

```diff
@@ -54,11 +54,15 @@
     "prettier": "^2.0.2",
     "spawn-command": "0.0.2-1",
     "ts-jest": "^26.5.6",
+    "tslib": "^2.2.0",
     "tslint": "^5.11.0",
     "typedoc": "^0.17.4",
     "typescript": "^4.2.4",
     "webpack-dev-middleware": "^3.6.0"
   },
+  "resolutions": {
+    "**/tslib": "^2.2.0"
```

Comment:
this ain't ideal but I couldn't make a more granular resolution (`rollup-plugin-typescript2/tslib`) to work, even though it should work according to the RFC for this feature